src/app/debug/table/table.component.ts

```diff
@@ -388,6 +391,13 @@ export class TableComponent implements OnInit, OnDestroy {
     }
   }
 
+  deleteAll(): void {
+    this.httpService.deleteAllReports(this.currentView.storageName).subscribe({
+      next: () => this.retrieveRecords(),
+      error: () => catchError(this.errorHandler.handleError()),
+    });
+  }
```

Comment:
I have just created a PR for this type of error checking. Please use a pipe instead of the error method here.

Here is an example of how I would like to see it being done:
```ts
this.httpService
    .getReport(storageId, this.currentView.storageName)
    .pipe(catchError(this.errorHandler.handleError()))
    .subscribe({
      next: (data: Report): void => {
        data.storageName = this.currentView.storageName;
        this.openReportEvent.next(data);
      },
    });
```